storage/pin/pin.go

```diff
@@ -41,39 +41,40 @@ var (
 	errInvalidUnmarshallData = errors.New("invalid data length")
 )
 
-// FileInfo is the struct that stores the information about pinned files
-// A map of this is stored in the state DB
-type FileInfo struct {
-	isRaw      bool
-	fileSize   uint64
-	pinCounter uint64
+// PinInfo is the struct that stores the information about pinned files
```

Comment:
should this just be Info. You know, no pkg prefix in names.
or call it `Pin`? maybe